src/components/business/Map.vue

```diff
@@ -33,7 +33,6 @@ export default {
   components: { MglMap, MglMarker },
   methods: {
     updateCoordinates (e) {
```

Comment:
Kept getting coordinate errors in the console, do I need a key?

Reply:
i think you do! I can put mine in secrets storage for you. End goal would be google-less but, y'know.